src/features/groups/components/GroupCardList/GroupCard.tsx

```diff
@@ -0,0 +1,29 @@
+import { Card } from '@/components/ui';
+import { fetchContainerList } from '@/lib/api/data';
+import { fetchUserList } from '@/lib/api/data';
+import { IContainer, IGroup, IUser } from '@/types/definition';
+
+import Link from 'next/link';
+
+import { ContainerCount } from './ContainerCount';
+import { UserCount } from './UserCount';
+
+export const GroupCard = async ({ group }: { group: IGroup }) => {
+  const containers: IContainer[] = await fetchContainerList(group.id);
+  const users: IUser[] = await fetchUserList(group.id);
+  return (
+    <Link href={`/groups/${group.id}`} key={group.id}>
```

Comment:
How about something like this?
You don't have to take all of this, just use for reference.

- `Link` should be  inside of `Card`.
- `span` instead of `p`
- Add `flex-col` for multiple GroupCard display.
```tsx
<Card asChild>
  <Link href={`/groups/${group.id}`} key={group.id} className="flex flex-col gap-3">
    <div className="w-full flex items-center justify-start">
      <span className="text-lg">{group.name}</span>
    </div>
    <div className="w-full flex justify-between">
      <ContainerCount containerCount={containers.length} />
      <UserCount userCount={users.length} />
    </div>
  </Link>
</Card>
```